test-suite/js/src/test/scala/org/scalajs/testsuite/library/StackTraceTest.scala

```diff
@@ -69,26 +69,34 @@ object StackTraceTest extends JasmineTest {
     when("nodejs").
     unlessAny("fullopt-stage", "strong-mode").
     it("decode class name and method name") {
-      verifyClassMethodNames("Foo" -> "f") {
-        new Foo().f(25)
-      }
+      val Error = js.constructorOf[js.Error]
+      val oldStackTraceLimit = Error.stackTraceLimit
+      Error.stackTraceLimit = 20
```

Comment:
When the optimizer is disabled, the new constructors in the chain cannot be eliminated, and caused one of the stack traces tested here to be longer. In particular, the `Baz.<init>` constructor *just* made it to the 11th depth. The test was failing because Node.js only records 10 frames by default.